### DISCUSSION TITLE
Array enum with exactly 1 enum does not render as a enum

### DISCUSSION THREAD

**Discussion by westmark** [scalar/scalar#1905] [Q&A]
Hi! I have schema that is an array of string enums. It only contains one enum. Currently it renders as an `array[] string` with no "available options".  If I add another enum, it renders as an `array[] string enum` with all available options included. This has to be a bug, right? Swagger renders a single enum array as expected.

(Yes an array with just 1 enum is pretty weird, but it is just the start. More will be added later)

**Comment by hanspagel** [comment]
That’s a bug, for sure! I’ve created an issue here: https://github.com/scalar/scalar/issues/2006
